Orm/Xtensive.Orm/Orm/Model/FieldInfo.cs

```diff
@@ -675,6 +675,7 @@ public override void Lock(bool recursive)
         column.Lock(true);
       if (associations.Count > 1) {
         var sorted = associations.Reorder();
+        associations.Clear();       // Unsubscribe to avoid memory leak
```

Comment:
Remove comment. I'll add description to merge comment so this info will be available in history but without chances to became misleading

Reply:
done